deploy/backplane/lpsre/02-cssre-monitoring-rolebinding.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
File should be renamed with lpsre instead of cssre.

Reply:
Oh yep thanks for the catch